src/pipelines/yolov8_classification/training_pipeline.py

```diff
@@ -36,36 +35,11 @@ def get_context() -> PicselliaContext:
 )
 def yolov8_classification_training_pipeline():
     # Data pipeline
-    dataset_context = data_extractor()
-    dataset_context = data_validator(dataset_context=dataset_context)
-    data_path = data_preparator(dataset_context=dataset_context)
-
-    # Model pipeline
-    weights_path = weights_extractor()
-    weights_path = weights_validator(weights_path=weights_path)
-    weights_path = weights_preparator(weights_path=weights_path)
-    model = model_loader(weights_path=weights_path)
-
-    # Training pipeline
-    callbacks = callback_preparator()
-    model = model_trainer(model=model, callbacks=callbacks, dataset_path=data_path)
-    exported_model_path = model_exporter(model=model)
-    _ = model_register(
-        weights_name="model-latest-onnx",
-        weights_path=str(exported_model_path),
-    )
-
-    # Evaluation pipeline
-    picsellia_predictions = model_inference(
-        model=model,
-        dataset_context=dataset_context,
-        attached_dataset_version="test",
-    )
-    _ = model_evaluator(
-        dataset_context=dataset_context,
-        attached_dataset_version="test",
-        picsellia_predictions=picsellia_predictions,
+    dataset_collection = training_data_extractor()
```

Comment:
I remove it because the steps callback/training/inference/evaluation steps wasn't develop correctly, there are part of future user stories for yolov8 classif. Do you want me to keep them?